google/cloud/storage/internal/default_client.h

```diff
@@ -37,8 +37,7 @@ class DefaultClient : public Client {
     // Assume the bucket name is validated by the caller.
     HttpRequestor requestor(storage_endpoint_ + "/b/" + request.bucket_name());
     requestor.AddWellKnownParameters(request.well_known_parameters());
-    requestor.AddHeader("Authorization: " +
-                        options_.credentials()->AuthorizationHeader());
+    requestor.AddHeader(options_.credentials()->AuthorizationHeader());
```

Comment:
So at after some ~misdirection~indirection, this ends up here:

https://github.com/GoogleCloudPlatform/google-cloud-cpp/blob/4dc8d2f8f1f2b6b06775a95921c156a16f91b5bb/google/cloud/storage/internal/curl_request.cc#L42

Do we need to change that function to ignore an empty `header` argument?

Reply:
Never mind, I was able to answer my own question.  It works, the empty header is ignored and has no effect on the request sent by curl.